packages/node/tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "outDir": "dist",
     "sourceMap": false,
     "declaration": true,
+    "skipLibCheck": true,
```

Comment:
When I added `edge-runtime` to this repo, I could no longer get `packages/node` to build because TypeScript was failing to process `*.d.ts` files inside of `@edge-runtime/primitives` and `@edge-runtime/vm`. Adding `skipLibCheck` here gets passed that error, but it seems like something we should keep enabled if possible.

```
~/source/vercel/vercel/packages/node $ yarn build

yarn run v1.22.18
warning package.json: "dependencies" has dependency "@vercel/ncc" with range "0.34.0" that collides with a dependency in "devDependencies" of the same name with version "0.24.0"
$ node build
../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1:46 - error TS2694: Namespace 'globalThis' has no exported member 'Crypto'.

1 declare class Crypto$1 implements globalThis.Crypto {
                                               ~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:2:18 - error TS2304: Cannot find name 'SubtleCrypto'.

2   public subtle: SubtleCrypto;
                   ~~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:7:47 - error TS2694: Namespace 'globalThis' has no exported member 'CryptoKey'.

7 declare class CryptoKey implements globalThis.CryptoKey {
                                                ~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:8:21 - error TS2304: Cannot find name 'KeyAlgorithm'.

8   public algorithm: KeyAlgorithm;
                      ~~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:10:16 - error TS2304: Cannot find name 'KeyType'.

10   public type: KeyType;
                  ~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:11:18 - error TS2304: Cannot find name 'KeyUsage'.

11   public usages: KeyUsage[];
                    ~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1231:16 - error TS2304: Cannot find name 'Blob'.

1231   Blob: typeof Blob
                    ~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1235:18 - error TS2693: 'Crypto' only refers to a type, but is being used as a value here.

1235   Crypto: typeof Crypto
                      ~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1237:24 - error TS2304: Cannot find name 'SubtleCrypto'.

1237   SubtleCrypto: typeof SubtleCrypto
                            ~~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1240:17 - error TS2304: Cannot find name 'fetch'.

1240   fetch: typeof fetch
                     ~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1241:16 - error TS2304: Cannot find name 'File'.

1241   File: typeof File
                    ~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1242:20 - error TS2304: Cannot find name 'FormData'.

1242   FormData: typeof FormData
                        ~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1243:19 - error TS2304: Cannot find name 'Headers'.

1243   Headers: typeof Headers
                       ~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1244:19 - error TS2304: Cannot find name 'Request'.

1244   Request: typeof Request
                       ~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1245:20 - error TS2304: Cannot find name 'Response'.

1245   Response: typeof Response
                        ~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1248:24 - error TS2304: Cannot find name 'CacheStorage'.

1248   CacheStorage: typeof CacheStorage
                            ~~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1249:17 - error TS2304: Cannot find name 'Cache'.

1249   Cache: typeof Cache
                     ~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1250:11 - error TS2304: Cannot find name 'CacheStorage'.

1250   caches: CacheStorage
               ~~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1315:23 - error TS2304: Cannot find name 'TextDecoder'.

1315   TextDecoder: typeof TextDecoder
                           ~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1316:23 - error TS2304: Cannot find name 'TextEncoder'.

1316   TextEncoder: typeof TextEncoder
                           ~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1323:15 - error TS2304: Cannot find name 'URL'.

1323   URL: typeof URL
                   ~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1324:27 - error TS2304: Cannot find name 'URLSearchParams'.

1324   URLSearchParams: typeof URLSearchParams
                               ~~~~~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1327:23 - error TS2304: Cannot find name 'WebAssembly'.

1327   WebAssembly: typeof WebAssembly
                           ~~~~~~~~~~~

../../node_modules/@edge-runtime/primitives/dist/index.d.ts:1357:45 - error TS1337: An index signature parameter type cannot be a union type. Consider using a mapped object type instead.

1357 declare function addPrimitives<T extends { [key: string | number]: any }>(
                                                 ~~~

../../node_modules/@edge-runtime/vm/dist/types.d.ts:6:6 - error TS1337: An index signature parameter type cannot be a union type. Consider using a mapped object type instead.

6     [key: string | number]: T;
       ~~~

../../node_modules/@edge-runtime/vm/dist/types.d.ts:14:6 - error TS1337: An index signature parameter type cannot be a union type. Consider using a mapped object type instead.

14     [key: string | number]: any;
        ~~~

../../node_modules/edge-runtime/dist/types.d.ts:18:28 - error TS2304: Cannot find name 'RequestInit'.

18     (input: string, init?: RequestInit): Promise<Response & {
                              ~~~~~~~~~~~

../../node_modules/edge-runtime/dist/types.d.ts:18:50 - error TS2304: Cannot find name 'Response'.

18     (input: string, init?: RequestInit): Promise<Response & {
                                                    ~~~~~~~~

src/dev-server.ts:188:28 - error TS2304: Cannot find name 'fetch'.

188     const response = await fetch(`${server.url}`, {
                               ~~~~~


Found 29 errors.

Error: Command failed: tsc
    at makeError (/Users/smassa/source/vercel/vercel/node_modules/execa/index.js:174:9)
    at /Users/smassa/source/vercel/vercel/node_modules/execa/index.js:278:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async main (/Users/smassa/source/vercel/vercel/packages/node/build.js:14:3) {
  code: 2,
  stdout: null,
  stderr: null,
  failed: true,
  signal: null,
  cmd: 'tsc',
  timedOut: false,
  killed: false
}
error Command failed with exit code 1.
```

I don't know exactly what's going on here. Any ideas?